packages/polywrap-uri-resolvers/polywrap_uri_resolvers/uri_resolver_wrapper.py

```diff
@@ -0,0 +1,36 @@
+from typing import Optional, Union, cast
+from polywrap_core import Uri, Client, IUriResolutionContext, UriPackageOrWrapper
+from polywrap_uri_resolvers import ResolverWithHistory
+from polywrap_result import Result, Ok, Err
+
+class UriResolverWrapper(ResolverWithHistory):
+    implementation_uri: Uri
+
+    def __init__(self, uri: Uri) -> None:
+        self.implementation_uri = uri
+
+    def get_step_description(self) -> str:
+        return super().get_step_description()
```

Comment:
Isn't this abstract? Why call super?

Reply:
this was autogenerated but the code editor, my bad

Reply:
actually i will just update this with returning a string and then, when the plugins are updated and this resolver is developed I will fix this